### PR TITLE
Create 1.12.0-alpha.0

### PR DESCRIPTION
**Pull request by howardjohn**
Get the first alpha build started. Mostly to test helm repo

### CODE REVIEW

File: release/trigger-build [hunk 1]
```diff
@@ -1 +1 @@
-1.10.0-alpha.0
```
Comment (mandarjog) [inline release/trigger-build:1]: Why a jump from 1.10 to 1.12 ?
Did we not follow the normal process before?

**Review by mandarjog — APPROVED** [review]
Lgtm

**Comment by ericvn** [comment]
A base image with vulnerabilities was detected, the new base images look to have been created and a PR to use them was as well. I've approved that PR so another spin should get the build going.

File: release/trigger-build (hunk 1 above)
Comment (howardjohn) [reply release/trigger-build:1]: Yeah we forgot to do a 1.11 alpha

**Pull request merged**